<?xml version="1.0"?>
<launch>
    <node pkg="collision_free_ik" type="main" name="main" output="screen">
        <param name="urdf_fp" type="string" value="$(find collision_free_ik)/config/panda.urdf"/>
        <param name="ee_frame" type="string" value="panda_hand_joint"/>
        <param name="arm_collider_fp" type="string" value="$(find collision_free_ik)/config/panda_collision.json"/>
        <param name="environment_fp" type="string" value="$(find collision_free_ik)/config/environment.json"/>
        <param name="initial_q" type="yaml" value="[0, 0, 0, 0, 0, 0, 0]"/>
    </node>
</launch>
